README.md

```diff
@@ -95,5 +95,15 @@ npm run build
 npm run test
 ```
 
+## Run a Registered Workflow
+ 
+ ```bash
+ git clone git@github.com:relaypro/relay-js.git
```

Comment:
Based on the changes I requested to keep sdk and samples separate (but in the same git repo), the series of commands would be...

```bash
git clone git@github.com:relaypro/relay-js.git
cd relay-js
npm install
npm run build
cd samples
npm install
node index.js
```